### PR TITLE
tiny bit more for python 3 migration

### PR DESCRIPTION
**Pull request by pieleric**

### CODE REVIEW

File: src/odemis/model/_vattributes.py [hunk 1]
```diff
@@ -988,9 +988,11 @@ def _check(self, value):
 
     def clip(self, val):
         """ Clip the given value to fit within the choices.
-
-        If the range contains vectors of length n, each element of val will be
-        clipped separately by position for positions 0..n-1
+          If the value is not within the choices the closest choice will be used.
+          If there is are "close" choice found, the current value is returned.
```
Comment (Sabrina85) [inline src/odemis/model/_vattributes.py]: Typo: `If there is a "close" choice found`

Comment (Sabrina85) [reply src/odemis/model/_vattributes.py]: What is the current value? Is not the "closest" found choice returned?

File: src/odemis/model/_vattributes.py [hunk 2]
```diff
@@ -1011,7 +1013,7 @@ def clip(self, val):
             if not ls:
                 return self.value  # in case of all choices of type tuple and not containing only numbers
 
-            return min(ls, key=lambda choice, distance: distance)[0]
+            return min(ls, key=lambda cd: cd[1])[0]
```
Comment (Sabrina85) [inline src/odemis/model/_vattributes.py]: What is cd now standing for :D

Comment (pieleric) [reply src/odemis/model/_vattributes.py]: `cd` stands for "Choice, Distance".

**Pull request merged**